cmd/wallet-web/src/pages/Credentials.vue

```diff
@@ -9,7 +9,7 @@
     <!-- Mobile Credentials Layout -->
     <div v-if="breakpoints.xs || breakpoints.sm" class="flex flex-col justify-start w-screen">
       <div class="bg-neutrals-white border border-neutrals-chatelle">
-        <flyout-menu />
+        <flyout-menu id="defaultMobileFlyoutMenu" />
```

Comment:
Can we give it more meaningful name? Something like `credentials-flyout-menu-mobile`